apps/protocol-frontend/src/components/DaoSettingsMembersTable.tsx

```diff
@@ -111,11 +114,61 @@ const DaoSettingsMembersTable = ({ daoId }: DaoSettingsMembersTableProps) => {
         header: 'Member Address',
         accessorKey: 'user',
         cell: ({ getValue }: { getValue: Getter<UIGuildUsers['user']> }) => {
-          return <Text whiteSpace="normal">{getValue().address}</Text>;
+          return getValue().address === userData?.address ? (
+            <Text
+              whiteSpace="normal"
+              fontWeight="bold"
+              bgGradient="linear(to-l, #7928CA, #FF0080)"
+              bgClip="text"
+            >
+              {getValue().address}
+            </Text>
+          ) : (
+            <Text whiteSpace="normal">{getValue().address}</Text>
+          );
+        },
+      },
+      {
+        header: 'Role',
+        accessorKey: 'membershipStatus',
+        cell: ({
+          getValue,
+        }: {
+          getValue: Getter<UIGuildUsers['membershipStatus']>;
+        }) => {
+          if (getValue() === undefined || getValue() === null) return;
+          return (
+            <Badge
+              minWidth="5rem"
+              textAlign="center"
+              bgColor={
+                getValue()?.name === 'Admin'
+                  ? 'brand.purple'
+                  : getValue()?.name === 'Member'
+                  ? 'brand.magenta'
+                  : 'gray.200'
+              }
+              color={
+                getValue()?.name === 'Admin'
+                  ? 'white'
+                  : getValue()?.name === 'Member'
```

Comment:
These don't need the ? because of the if statement above

Reply:
I thought so as well since we have that check but removing the optional chaining does cause an issue about getValue().name possibly being undefined or null.

Reply:
I am guessing it is because the getValue function is being called under the if statement. My guess is the typing would remove the need for optional chaining if it was abstract to a variable and accessed via dot notation

Reply:
That's a good idea. I think so too, since it seemed that the function call was ignoring the type check.